### ISSUE TITLE
inconsistent content of model catalog entries

### ISSUE THREAD

**Issue by mmigliore** [cnr-ibf-pa/hbp-bsp-issues#289]
### Expected behavior
entries using the same model must refer to the same cell type.
### Actual Behavior
different entries using an interneuron model report either interneuron or pyramidal as cell type. 
For example:
![image](https://user-images.githubusercontent.com/28526575/39964236-6b88e056-5680-11e8-98a6-fc88bdd9a175.png)

### Steps to reproduce the problem
go to model catalog

**Comment by appukuttan-shailesh** [comment]
These entries seem to have come up during usecase testing. We have decided to make certain alterations to the validation usecases which will help prevent duplication of model entries in future. As these changes would require the usecases to be tested anew, we have kept them for post SGA1 review. For now these spurious entries have simply been deleted.

p.s. we will also incorporate appropriate provisions to handle "testing" data on the validation framework.
